its/ruling/src/test/resources/sonar-server/java-S4165.json

```diff
@@ -1,4 +1,11 @@
 {
+'org.sonarsource.sonarqube:sonar-server:src/main/java/org/sonar/server/es/textsearch/ComponentTextSearchQueryFactory.java':[
+177,
+178,
+179,
+180,
+181,
+],
```

Comment:
For the record, the different issues additions/deletion in this PR (like these ones) correspond to issues removed in #3140.

Reply:
Yep, I have seen that -_-